miner/worker.go

```diff
@@ -81,6 +83,14 @@ const (
 	staleThreshold = 7
 )
 
+var (
+	// Metrics for the skipped txs
+	gasLimitExceededMeter       = metrics.NewRegisteredMeter("miner/skipped_txs/gas_limit_exceeded", nil)
+	rowConsumptionOverflowMeter = metrics.NewRegisteredMeter("miner/skipped_txs/row_consumption_overflow", nil)
```

Comment:
discuss with @HAOYUatHZ , with can classify this metric into `l1TxRowConsumptionOverflow` and `l2TxRowConsumptionOverflow`.